src/components/alert/index.jsx

```diff
@@ -0,0 +1,7 @@
+
+export const Alert = ({alertMessage, backgroundColor }) => {
+    const boxStyles = `block w-full text-white ${backgroundColor} text-center font-semibold p-4 mb-3`
+    return (
+          <div className={boxStyles}>{alertMessage}</div>
```

Comment:
In order to be consistent (because creating a variable that stores the classes for an html element is not something that we are making use of in the project) move the expression to the JSX className={`...`}